### PR TITLE
Tests: Add functional tests

### PR DESCRIPTION
**Pull request by bethge**
Adds simple functional tests for the w3c test suite.

### CODE REVIEW

File: tests/functional/pointerevent_pointerover-manual.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  var name = 'pointerevent_pointerover-manual';
+
+  registerSuite({
+    name: name,
+
+    main: function() {
+      return w3cTest(this.remote, name + '.html')
+        .findById('target0')
+          .moveMouseTo(50, 25)
+          .moveMouseTo(50, -25)
```
Comment (scottgonzalez) [inline tests/functional/pointerevent_pointerover-manual.js]: This second move isn't necessary for the test.

File: tests/intern.js [hunk 2]
```diff
@@ -51,8 +52,61 @@ define({
...
+    // 'tests/functional/pointerevent_pointerleave_pen-manual',
+    // 'tests/functional/pointerevent_pointerleave_touch-manual',
 
-    'tests/functional/pointerevent_pointermove_isprimary_same_as_pointerdown-manual'
+    // 'tests/functional/pointerevent_pointermove-manual',
+    // 'tests/functional/pointerevent_pointermove-on-chorded-mouse-button',
+    'tests/functional/pointerevent_pointermove_isprimary_same_as_pointerdown-manual',
+     'tests/functional/pointerevent_pointermove_pointertype-manual',
+
+     'tests/functional/pointerevent_pointerout-manual',
```
Comment (scottgonzalez) [inline tests/intern.js]: Bad indentation (the line above too).

File: tests/intern.js [hunk 3]
```diff
@@ -51,8 +52,61 @@ define({
...
+
+    // 'tests/functional/pointerevent_pointerout_after_pointercancel_touch-manual.js',
+    // 'tests/functional/pointerevent_pointerout_after_pointerup_nohover-manual.js',
+    // 'tests/functional/pointerevent_pointerout_pen-manual.js',
+
+     'tests/functional/pointerevent_pointerout_received_once-manual.js',
+     'tests/functional/pointerevent_pointerover-manual.js',
+     'tests/functional/pointerevent_pointertype_mouse-manual.js',
```
Comment (scottgonzalez) [inline tests/intern.js]: These too.

File: tests/intern.js [hunk 4]
```diff
@@ -51,8 +52,61 @@ define({
...
+
+    // 'tests/functional/pointerevent_pointertype_pen-manual.js',
+    // 'tests/functional/pointerevent_pointertype_touch-manual.js',
+
+     'tests/functional/pointerevent_pointerup-manual.js',
+     'tests/functional/pointerevent_pointerup_isprimary_same_as_pointerdown-manual.js',
+     'tests/functional/pointerevent_pointerup_pointertype-manual.js'
```
Comment (scottgonzalez) [inline tests/intern.js]: And these.

**Comment by bethge** [comment]
Thanks for the review. I updated according. I just noticed a typo in: https://github.com/jquery/PEP/pull/256/files#diff-9d42abc2f38cbe5e7020185c7d511dccR4 
which made me insert a quite long sleep:
https://github.com/jquery/PEP/pull/256/files#diff-9d42abc2f38cbe5e7020185c7d511dccR14 due to:
https://github.com/w3c/web-platform-tests/blob/master/pointerevents/pointerevent_pointerout_received_once-manual.html#L47

**Pull request closed without merging**